rllib/tuned_examples/sac/pendulum_sac_envrunner.py

```diff
@@ -0,0 +1,59 @@
+import ray
+
+from ray import air, tune
+from ray.rllib.algorithms.sac.sac import SACConfig
+from ray.rllib.env.single_agent_env_runner import SingleAgentEnvRunner
+
+config = (
+    SACConfig()
+    # Enable new API stack and use EnvRunner.
+    .experimental(_enable_new_api_stack=True)
+    .rollouts(
+        rollout_fragment_length=1,
+        env_runner_cls=SingleAgentEnvRunner,
+        num_rollout_workers=1,
+    )
+    .environment(env="Pendulum-v1")
+    .training(
+        lr=3e-4,
+        target_entropy="auto",
+        # TODO (simon): Implement n-step.
+        n_step=1,
+        tau=0.005,
+        train_batch_size=256,
+        target_network_update_freq=1,
+        replay_buffer_config={
+            "type": "EpisodeReplayBuffer",
+        },
+        num_steps_sampled_before_learning_starts=256,
+        model={
+            "initial_alpha": 1.001,
```

Comment:
Ah, I see :) here is the `initial_alpha` (directly defined inside the model config).

Let's make a TODO that we have to revisit this problem: config properties that the Module must know about need to be written to the `model_config_dict` automatically.

Reply:
Oh sorry, that one is obsolete. That was before I used the `nn.Parameter` directly inside of the `TorchLearner._get_tensor_variable`. Let's behave as if it is not there. I will remove it in the next commit.

Reply:
We have the `initial_alpha` now used in the `SACLearner.build()` when `SACLearner.curr_log_alpha` is defined.